tests/e2e/README.md

```diff
@@ -121,6 +121,8 @@ codebase.
 
 ## `upgrade` Package
 
+***This package has been removed and it's logic has been moved to `initialization` package***
```

Comment:
Do you mind updating the references to refer to the right existing package and remove references to the non-existent package please?